SQL/0000-00-00-schema.sql

```diff
@@ -439,6 +438,87 @@ CREATE TABLE `Visit_Windows` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
+-- ********************************
+-- tarchive tables
+-- ********************************
+
+
+CREATE TABLE `tarchive` (
+  `DicomArchiveID` varchar(255) NOT NULL default '',
```

Comment:
Suggest to put primary key as first field and ordering field by type for efficiency: int, datetime, double, varchar, text

Reply:
taught it was a new table. forget for now would need major cleanup